core/controllers/dashboard.py

```diff
@@ -257,6 +258,204 @@ def _round_average_ratings(rating):
         self.render_json(self.values)
 
 
+class LearnerDashboardPage(base.BaseHandler):
```

Comment:
(It might be simpler to do the renaming of the existing stuff in a separate PR, though, that gets merged into develop before this one.)